### DISCUSSION TITLE
How do I use C-h with meow's keypad mode?

### DISCUSSION THREAD

**Discussion by jgarte** [meow-edit/meow#660] [Q&A] [answered]
Hi,

How do I use C-h with meow's keypad mode to determine what elisp function was called when I run a keypad command?

**Comment by DogLooksGood** [comment]
Use `meow-keypad-describe-key`. It's a keypad version for  `C-h k`.

**Reply by jgarte** [reply]
I did `SPC-/ SPC-x 1` but that doesn't do what I expect which is to show the docs for `delete-other-windows`.

`SPC-/` is bound to `meow-keypad-describe-key`.

WDYT

**Reply by DogLooksGood** [reply, marked as answer]
You should do `SPC / x 1`. `meow-keypad-describe-key` will also take you to the keypad state so no second `SPC`.

**Reply by jgarte** [reply]
Thanks!
